src/Bot/commands/externals/gpt.ts

```diff
@@ -0,0 +1,28 @@
+import { Configuration, OpenAIApi } from 'openai'
+import { config } from '../../../../config'
+
+const configuration = new Configuration({
+    apiKey: config.openaiApiKey,
+})
+const openai = new OpenAIApi(configuration)
+
+export const sendPrompt = async (question: string) => {
+    const prompt = `You\'re a coding assistant. You help students and explain things to them in a simple, clear, and super-friendly way, and you only respond to questions related to coding or programming.\n\n###EXAMPLE\n\nStudent: What's const in JavaScript?\nCoding assistant: \n> What's const in JavaScript?\n\n"const" keyword in JavaScript is a way to declare variables with a value that'll never change.\n\n###CURRENT\n\nStudent: ${question}\nCoding assistant:`
+    const response = await openai.createCompletion({
+        model: 'text-davinci-003',
+        prompt,
+        temperature: 0.7,
+        max_tokens: 256,
```

Comment:
Try a larger max token count, the prompt is already >100 tokens, combined with the user's question this only leaves half tokens left to fit the answer in.